backend/hardhat.config.ts

```diff
@@ -260,6 +260,91 @@ task('setGaslessKeyPair')
     console.log(`Successfully set gasless keypair to ${args.payerAddress}, secret ${args.payerSecret} and nonce ${nonce}. Transaction hash: ${receipt!.hash}`);
   });
 
+// Deploy and setup Quiz contract
+task('deployAndSetupQuiz', 'Deploys Quiz contract and performs setup tasks')
+  // .addOptionalParam('network', 'Specify the network', 'sapphire-testnet')
+  .addOptionalParam('questionsfile', 'File containing questions in JSON format', 'test-questions.json')
+  .addOptionalParam('couponsfile', 'File containing coupons, one per line', 'test-coupons.txt')
+  .addOptionalParam('reward', 'Reward in ROSE', '2.0')
+  .addOptionalParam('gaslessaddress', 'Payer address for gasless transactions')
+  .addOptionalParam('gaslesssecret', 'Payer secret key for gasless transactions')
+  .addOptionalParam('fundamount', 'Amount in ROSE to fund the contract', '100')
+  .addOptionalParam('fundgaslessamount', 'Amount in ROSE to fund the gasless account', '10')
+  .addOptionalParam('contractaddress', 'Contract address for status check')
+  .setAction(async (args, hre) => {
+    // Run compilation
+    await hre.run('compile');
+
+    // For deployment unwrap the provider to enable contract verification.
+    const uwProvider = new JsonRpcProvider(hre.network.config.url);
+    // const accounts = await hre.ethers.getSigners();
+
+    // Deploy Quiz contract
```

Comment:
Add full stops at the end of one-liner comments ;)

```suggestion
    // Deploy Quiz contract.
```

Reply:
Updated the comments and removed unnecessary leftover code.